uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/CRM/Tasksassignments/Hook/PageRun/ResourcesTest.php

```diff
@@ -6,38 +6,47 @@
  * @group headless
  */
 class CRM_Tasksassignments_Hook_PageRun_ResourcesTest extends BaseHeadlessTest {
+  /**
+   * @var string
+   *   The key for the T&W extension.
+   */
+  const EXTENSION_KEY = 'uk.co.compucorp.civicrm.tasksassignments';
```

Comment:
Because you defined this constant in another class you can just reference it here instead of redefining it.

Constants are public and can be referenced like `MyClass::MY_CONSTANT`